app/javascript/dashboard/components/ChatList.vue

```diff
@@ -346,6 +346,9 @@ export default {
       if (!this.hasAppliedFilters) {
         this.fetchConversations();
       } else {
+        this.appliedFilters[
+          this.appliedFilters.length - 1
+        ].query_operator = null;
```

Comment:
Is there a way to remove the query operator when it's applied?

Reply:
You mean when making the api request?

Reply:
No, Like the applied filter object shouldn't coming with the non empty `query_operator` in the first place I guess. That would be the root cause of this problem. 

@iamsivin  also came across this issue in another place, So fixing the root would be the best case scenario.

Reply:
Possibly in `app/javascript/dashboard/components/widgets/conversation/ConversationAdvancedFilter.vue`

Reply:
@nithindavid We need to remove this manually because there is no way of knowing if a filter appended in the list would be the last unless we click submit.

But I have moved removing this operator to the payload generator helper method in https://github.com/chatwoot/chatwoot/pull/3747/commits/18e10fc6a8e824ab6e4e3256c037bee9a7642509, so this has to be never done manually in our logic. Is be this okay?